test/test_popup.py

```diff
@@ -26,24 +26,24 @@
 from test.conftest import BareConfig
 
 
-@pytest.mark.parametrize("qtile", [BareConfig], indirect=True)
-def test_popup_focus(qtile):
-    qtile.test_xeyes()
-    qtile.windows_map = {}
+@pytest.mark.parametrize("m", [BareConfig], indirect=True)
+def test_popup_focus(m):
+    m.test_xeyes()
+    m.windows_map = {}
 
-    # we have to add .conn so that Popup thinks this is libqtile.qtile
-    qtile.conn = xcbq.Connection(qtile.display)
+    # we have to add .conn so that Popup thinks this is libqtile.m
```

Comment:
Search and replace worked too well on this one. :)

Reply:
Lol yep